src/mjolnir/ingest_transit.cc

```diff
@@ -140,10 +144,12 @@ uint32_t to_local_epoch_sec(const std::string& dt) {
   return static_cast<uint32_t>(tp.time_since_epoch().count());
 };
 
-// Read from GTFS feed, sort data into the unique tiles they belong to
-std::priority_queue<tile_transit_info_t> select_transit_tiles(const boost::property_tree::ptree& pt) {
+std::string get_origin_id_base(const std::string& stop_id, const std::string feed_name) {
+  return feed_name + "_" + stop_id;
+}
 
-  auto path = pt.get<std::string>("mjolnir.transit_feeds_dir");
+// Read from GTFS feed, sort data into the unique tiles they belong to
+std::priority_queue<tile_transit_info_t> select_transit_tiles(const std::string& gtfs_path) {
```

Comment:
we pass in the path to the gtfs directory, since we had to get that anyways in the calling function. also ensures to have a trailing separator, making things a bit safer.